### PR TITLE
tests: refactor

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

tests

**Did you add tests for your changes?**
yes

**If relevant, did you update the documentation?**
no

**Summary**
- refactor test, no need output setting

**Does this PR introduce a breaking change?**
no

**Other information**
no

### CODE REVIEW

**Comment by anshumanv** [comment]
@evilebot @alexander-akait is this what you had in mind?

File: test/utils/test-utils.js [hunk 1]
```diff
@@ -29,18 +29,15 @@ const hyphenToUpperCase = (name) => {
  *
  * @param {String} testCase The path to folder that contains the webpack.config.js
  * @param {Array} args Array of arguments to pass to webpack
- * @param {Boolean} setOutput Boolean that decides if a default output path will be set or not
  * @param {Array<string>} nodeOptions Boolean that decides if a default output path will be set or not
  * @param {Record<string, any>} env Boolean that decides if a default output path will be set or not
  * @returns {Object} The webpack output or Promise when nodeOptions are present
  */
-const run = (testCase, args = [], setOutput = true, nodeOptions = [], env) => {
+const run = (testCase, args = [], nodeOptions = [], env) => {
```
Comment (alexander-akait) [inline test/utils/test-utils.js:36]: `nodeOptions` and `env` should be merged into `options` and let's use execa, no need `spawnSync`

Comment (anshumanv) [reply test/utils/test-utils.js:36]: yep no need

Comment (anshumanv) [reply test/utils/test-utils.js:36]: wip

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Freeze on github actions, but all works fine

**Pull request merged**